weather.py

```diff
@@ -0,0 +1,306 @@
+# Imports written in alphabetical order
+from os import path
+import sys
+from termcolor import colored
+import zipfile
+
+
+class Weatherman:  # Weatherman Application in one class
+
+    file_pre = "Murree_weather_"  # The part of the file name that is same for all files
```

Comment:
When we are working on actual projects, the readability of code is very important because we write code once and read it every time we need to add a new feature or fix a bug. Using proper names increases readability by a large factor. It will be hard to perceive its intention in the code and to get a clear understanding we will have to find its initialization. If using longer names makes the intention clear then it's totally fine to use longer names.
```suggestion
    file_name_prefix = "Murree_weather_"  # The part of the file name that is same for all files
```

Reply:
change made as suggested.